src/static/css/product-card.scss

```
@@ -0,0 +1,197 @@
+
+            .faz {
+                color: $line-bgcolor; 
+                font-size: 10px;
+            }
+
+            .review {
+                color: $description-color;
+                display: inline-block;
+                font-size: 14px;
+            }
+        }
+
+        .in-stock {
+            color: $description-color;
+            font-size: 12px;
+            line-height: 1;
+
+            .fas {
+                color: $circle-green;
+            }
+
+            .circle {
+                color: $title-color;
+            }
+        }
+
+        .description {
+            color: $description-color;
+            font-size: 16px;
+            line-height: 2.1;   
+            margin-top: 0;
+            
+        }
+    }
+}
+
+.add-cart {
+    display: block;
+
+    .add-btn {
+        align-items: center;
+        display: flex;
+        margin-bottom: 50px !important;
+
+        .qty {
+            background-color: $quantity-background;
+            line-height: 40px;
+            margin: 0;
+            padding-left: 15px;
+            padding-right: 15px;
+        }
+
+        .quantity {
+            position: relative;
+
+            .qty-number {
+                background-color: $quantity-background;
+                border: none;
+                color: $quantity-color;
+                font-family: $font-stack;
+                height: 40px;
+                padding: 0px;
+                text-align: center;
+                width: 100px;
+                
+                &{
+                    margin: 0;
+                    -moz-appearance:textfield;
+                    -webkit-appearance: textfield;
+                }
+            }
+
+            .qty-more {
+                color: $quantity-color;
+                cursor: pointer;
+                height: 20px;
+                line-height: 30px;
+                position: absolute;
+                right: 0;
+                top: 0;
+                text-align: center;
+                width: 30px;
+            }
+
+            .qty-less {
+                cursor: pointer;
+                color: $quantity-color;
+                height: 25px;
+                line-height: 1px;
+                position: absolute;
+                right: 0;
+                text-align: center;
+                top: 21px;
+                width: 30px;
+            }
+
+        }
+    }
+    
+    .button {
+        background-color: $line-bgcolor;
+        border: 0;
+        color: $moramia-white;
+        font-family: $font-stack;
+        font-size: 18px;
+        font-weight: 400;
+        height: 80px;
+        line-height: 80px;
+        padding-bottom: 0;
+        padding-left: 7px;
+        padding-right: 7px;
+        padding-top: 0;
+        width: 310px;
+        
+        &:hover {
+            background-color: $background-button;
+            color: $moramia-white;
+            cursor: pointer;
+            font-size: 18px;
+        }
```

```suggestion
        }
        &:focus {
            background-color: $background-button;
            border: solid 2px $moramia-blue;
        }
```
you could put a focus element to help accessibility semantics